scripts/scr_inventory_step.gml

```diff
@@ -0,0 +1,25 @@
+if (active)
+{
+    /********************/
+    // Temporary Controls
+    /********************/
+    keyUp = keyboard_check_pressed(ord('W'));
+    keyDown = keyboard_check_pressed(ord('S'));
+    keyLeft = keyboard_check_pressed(ord('A'));
+    keyRight = keyboard_check_pressed(ord('D'));
+    keyAction = keyboard_check_pressed(ord('E'));
+
+    // Update selection
```

Comment:
I don't think these comments are worth much, I think you can just read the code and understand what's going on.

You could do the following block as this:
```
selectX += (keyRight - keyLeft) % ELEM_PER_ROW;
selectY += (keyDown - keyUp) % ELEM_PER_COL;
```
where ELEM_PER_ROW and ELEM_PER_COL are defined constants.
I think this would be cleaner, will drop the constant numbers, and won't require the comments probably.

Reply:
fair point